packages/experience-components/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "@silverhand/ts-config-react/tsconfig.base",
+  "compilerOptions": {
+    "baseUrl": "./",
+    "outDir": "./lib",
+    "paths": {
+      "@/*": ["./src/*"]
```

Comment:
don't think this is a good idea for components. it may cause issues when importing from another proejct